docusaurus/docs/Android/02-client/06-guides/01-push-notifications/01-overview.mdx

```diff
@@ -18,7 +18,7 @@ To receive push notifications from Stream Chat, you'll need to:
 We support the following providers:
 
 - [Firebase](./02-firebase.mdx)
-- Huawei Push Kit (Coming soon, stay tuned!)
+- [Huawei Push Kit](./03-huawei.mdx)
```

Comment:
Leaving this on a random line: we should update the changelog to mention this new module

Reply:
DONE ;)